readme.md

```diff
@@ -8,3 +8,6 @@
 * Install .NET Core 3.0 SDK, https://dotnet.microsoft.com/download/dotnet-core/3.0
 * `cd Armadillo.Server`
 * `dotnet run`
+
+To run in Linux environment install `gss-ntlmssp` package.
+Specify your username and password in `Armadillo.Agent` and `Armadillo.Server` JSON files.
```

Comment:
Let's split it into two parts - the one related to the agent and the one related to the server. Agent readme is here https://github.com/skalitin/Armadillo/blob/master/Armadillo.Agent/readme.md
Also it is not clear what are 'JSON files'. let's use dotnet user-secrets command